pkgs/by-name/wa/wavelog/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nix-update-script,
+  php,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "wavelog";
+  version = "1.9.1";
+
+  src = fetchFromGitHub {
+    owner = "wavelog";
+    repo = pname;
+    tag = version;
+    hash = "sha256-BYCRqb27QWOo74w3O6tfZGEDF3UInsgshsIm9uxOm+8=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/
+    ln -s $src $out
```

Comment:
that will create a sdjfklasjfdalkfjdskla-source into $out, you should remove the mkdir for it to work if you use the ln approach

Reply:
Hm yeah this creates a bunch of chmod errors.